wcm-io-qa/wcm-io-qa-galenium
# Feature/junitrunner

How to use JUnit for faster test feedback:

1. add this dependency to the project with the galenium tests
```
<dependency>
  <groupId>io.wcm.qa</groupId>
  <artifactId>io.wcm.qa.galenium.junitrunner</artifactId>
  <version>1.0.0-SNAPSHOT</version>
  <scope>test</scope>
</dependency>
```

2. add this annotation to the test class (or any parent of you test class)
@ExtendWith(GaleniumExtension.class)

3. add this annotation to the test method
@TestTemplate

4. run the test


How to use JUnit to generate example texts:

1. Add the annotation @Sampling to your test class

2. run the test

3. check your expected.properties
